frontend/containers/open-call-page/header/component.tsx

```diff
@@ -0,0 +1,190 @@
+                            {type}
+                          </p>
+                        );
+                      })}
+                    </div>
+                    <span id="open-call-instrument-types" className="text-gray-400">
+                      <FormattedMessage
+                        defaultMessage="Instrument {numInstrumentTypes, plural, one {type} other {types}}"
+                        id="OTKLo8"
+                        values={{
+                          numInstrumentTypes: instrument_types?.length || 0,
+                        }}
+                      />
+                    </span>
+                  </div>
+                </div>
+                <hr className="mt-6 mb-8" />
+                <div className="flex flex-col justify-between pl-2 sm:flex-row">
+                  <div className="flex flex-col items-center justify-end w-full gap-2 text-center sm:w-1/3">
+                    <div className="w-[82px] h-[82px] rounded-full flex justify-center items-center">
+                      <OpenCallChart openCallRange={openCallRange} />
+                      <p className="absolute max-w-[62px] text-xs font-semibold text-green-dark">
+                        {status === OpenCallStatus.Launched ? (
+                          <FormattedMessage defaultMessage="Ending soon" id="8mCCtS" />
+                        ) : (
+                          <FormattedMessage defaultMessage="Closed" id="Fv1ZSz" />
+                        )}
+                      </p>
+                    </div>
+                  </div>
+                  <div className="flex flex-col items-center justify-end w-full gap-2 sm:w-2/3 sm:items-end">
+                    <span id="total-of-projects" className="text-2xl font-semibold text-gray-700">
+                      {openCallRange.deadline}
+                    </span>
+                    <span aria-labelledby="total-of-projects" className="text-gray-400">
+                      <FormattedMessage defaultMessage="Deadline" id="8/Da7A" />
+                    </span>
+                  </div>
+                </div>
+              </>
+            )}
+            {/* <div className="flex flex-col justify-between gap-4 lg:flex-row mt-7">
+              <Button
+                className="justify-center"
+                theme="secondary-green"
+                onClick={handleFavorite}
+                // disabled={!user || favoriteLoading}
+                aria-pressed={isFavorite}
+              >
+                <Icon icon={Heart} className={cx('w-4 mr-3', { 'fill-green-dark': isFavorite })} />
+                <FormattedMessage defaultMessage="Favorite" id="5Hzwqs" />
+              </Button>
+              <Button
+                className="w-full lg:max-w-[200px] justify-center"
+                theme="primary-green"
+                onClick={handleApply}
+              >
+                <FormattedMessage defaultMessage="Apply now" id="VR4TEV" />
+              </Button>
+            </div> */}
```

Comment:
Can you show this section? You can disable the buttons while we don't have the features implemented.